src/components/switch/Switch.constants.ts

```diff
@@ -0,0 +1,25 @@
+import {
```

Comment:
nit: to have a better structure it would be better to group the arrays with the default values.